src/common/src/util/value_encoding/mod.rs

```diff
@@ -33,10 +35,21 @@ use crate::types::{
 
 pub mod error;
 use error::ValueEncodingError;
+
+use self::column_aware_row_encoding::ColumnAwareSerde;
 pub mod column_aware_row_encoding;
 
 pub type Result<T> = std::result::Result<T, ValueEncodingError>;
 
+/// The kind of all possible `ValueRowSerde`.
```

Comment:
If we run fast enough, perhaps we could remove the `Basic` encoding in 0.18. (Because the storage's side work has break the compatibility anyway. Of course just for this time 🥵)

Reply:
I'm thinking of still using the basic value-encoding for the tables that (temporarily) not supports schema change, like internal tables or normal materialized views (except for Index). The reason is that "column ID" does not make much sense for these tables with immutable schema, and we can save more storage costs.